frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pageComponents/PreviewDemand.js

```
@@ -0,0 +1,262 @@
+    </div>
+);
+const InputFieldNew = ({ label, value }) => (
+    <div style={styles.field}>
+        <div style={styles.label}>{label}</div>
+        <input style={styles.inputs} value={value} readOnly />
+    </div>
+);
+const PropertyForm = () => {
+    return (
+        <Card>
+            <button style={styles.downloadBtn}>⬇ Download</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)